### PR TITLE
New user creation, display existing user error, and some cleanup.

### PR DESCRIPTION
**Pull request by cotey-gh**
Send new user to backend for creation; 
Display existing user Error; 
Fix control.getValues error; 
Add type to InputField for uses like to obscure password by setting to type="password";
Change autocomplete from current-password to new-password;

### CODE REVIEW

File: src/components/SignUp.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+// import React from 'react';
```
Comment (ConnorDBurge) [inline src/components/SignUp.js:1]: Can you got through and clean up the comments?

File: src/components/SignUp.js [hunk 2]
```diff
@@ -63,12 +65,33 @@ const useStyles = makeStyles((theme) => ({
...
+  const { handleSubmit, control, formState: { errors }, getValues } = useForm();
 
-  const onSubmit = (data) => {
-    // Additional actions on form submission here
-    // placemarker to call our API to chk and save data
-    console.log(data);
+  const [errorMessage, setErrorMessage] = useState(null);
+  
+  const onSubmit = async (data) => {
+    // console.log(data);
```
Comment (ConnorDBurge) [inline src/components/SignUp.js:73]: Lets clean this up.

**Review by ConnorDBurge — APPROVED** [review]

**Comment by ConnorDBurge** [comment]
Everything looks good. I pulled it down to test it out. Lets make sure we redirect to the task page on a successful sign up (status 200). The user should be logged in and redirected to the tasks route.

**Comment by cotey-gh** [comment]
@ConnorDBurge Does the SignUp page redirect to the Tasks page or SignIn page? 

I thought it was the SignIn page, so I was waiting for @hrostaiee to push the updated SignIn page before redirecting to it. 

The SignUp page is not doing any Auth (e.g., obtaining accessToken, storing token, etc.).

**Comment by hrostaiee** [comment]
It's the sign in page

File: src/components/SignUp.js (hunk 2 above)
Comment (cotey-gh) [reply src/components/SignUp.js:73]: I'll go through and check for any unneeded comments/commented out stuff when I do the redirect (to SignIn?).

**Comment by cotey-gh** [comment]
@ConnorDBurge I cleaned up the comments and anything else I had commented out for testing etc.

Since the CMSC495-27 branch also has some changes to `SignUp.js`, I am going to wait until @hrostaiee finishes with CMSC495-27 and merges though before any commit and push to avoid any potential merge conflicts.

**Pull request merged**